lua/refactoring/tests/extract.simple-function.expected.lua

```diff
@@ -1,14 +1,13 @@
-
 local function foo_bar(a, test, test_other)
-        for idx = test - 1, test_other do
+    for idx = test - 1, test_other do
```

Comment:
next time avoid doing to much of a refactor.  these could of remained, make a ticket, and we could of resolved it shortly thereafter.

Reply:
It is due to formatting, I havent touched those files